Platform.PCL/Realm.Sync.PCL/Exceptions/HttpException.cs

```diff
@@ -21,12 +21,28 @@
 
 namespace Realms.Sync
 {
+    /// <summary>
+    /// An exception thrown when a transport error occurs during authentication. 
+    /// </summary>
     public class HttpException : Exception
     {
+        /// <summary>
+        /// Gets the <see cref="HttpStatusCode"/> of the response.
+        /// </summary>
         public HttpStatusCode StatusCode { get; }
 
+        /// <summary>
+        /// Gets the ReasonPhrase of the response.
+        /// </summary>
         public string ReasonPhrase { get; }
 
+        /// <summary>
+        /// Gets the body of the response.
```

Comment:
How is this helpful?

IMO it's better to have a more descriptive member name than having to explain an obscure one.